NestJS/src/Event/calendar/calendar_credentials.json

```diff
@@ -0,0 +1 @@
+{"access_token":"ya29.GlvzBY-L13Gh7v294vZTkvWFsWeV8fHPxSzx5SH47zsoIxRGRMQIn91U66e1euzw2UTHQYQ0bNPN9saOGYBZGng7T_YUV8BsDtefTUsBduqgpFbI9UVUPOWHfShl","token_type":"Bearer","refresh_token":"1/eoS_9a-AJNx0pkkwaLQtT0pA6j1o0m4xD80XqIDipbU","expiry_date":1531132335139}
```

Comment:
please dont commit such info into git

Reply:
Sorry, I forgot to put it to gitignore. I have already put it to gitignore

Reply:
please delete this file from PR